app/assets/stylesheets/mobile/login-modal.scss

```diff
@@ -1,292 +0,0 @@
-.static-login {
-  #main-outlet-wrapper,
-  #main-outlet {
-    padding: 0;
-  }
-
-  .contents.body-page {
-    margin: 0;
-  }
-}
-
-.login-welcome {
```

Comment:
Same with `.login-welcome`, it is used in `templates/login.gjs`.

Reply:
We have `app/assets/stylesheets/common/base/static-login.scss` that seems to cover thius